### PR TITLE
feat(addon-commerce): add length check

### PR DESCRIPTION
**Pull request by Smirnov906**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [ ] No

### CODE REVIEW

File: projects/addon-commerce/utils/is-card-length-valid.ts [hunk 1]
```diff
@@ -20,6 +20,6 @@ export function isCardLengthValid(cardNumber: string): boolean {
         case TuiPaymentSystem.Visa:
             return length > 12 && length < 20;
         default:
-            return false;
+            return length > 8;
```
Comment (splincode) [inline projects/addon-commerce/utils/is-card-length-valid.ts:23]: Could you also add tests that fix this bug?

Comment (Smirnov906) [reply projects/addon-commerce/utils/is-card-length-valid.ts:23]: Add in new commit

Comment (splincode) [reply projects/addon-commerce/utils/is-card-length-valid.ts:23]: What do you think about?

`return length > 8 && length < 20;`

image

**Review by splincode — APPROVED** [review]
@waterplea please, check it :)

**Review by waterplea — APPROVED** [review]

**Pull request merged**